robots/flakefinder/downloader.go

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
`The Kubernetes Authors`? Is this code a copy from some k8s repo? If not then it should be `Kubevirt` :)

Reply:
Done.